sympy/calculus/util.py

```diff
@@ -10,6 +10,131 @@
 from sympy.functions.elementary.miscellaneous import Min, Max
 
 
+def continuous_in(f, symbol, interval):
+    """
+    Finds the intervals of continuity of a function in a given interval range.
```

Comment:
Initially, I thought the same. In my initial commits, I had named the argument `domain` instead of `interval`. However, I realised that this naming could be misleading at times.
We are not always interested in working with the entire domain of a function.
Sometimes, the sub-domains are relevant to us. 
So, I chose to use a general term `interval` instead of `domain`.